### PR TITLE
Fixing the camel case setting for getters and setters

### PR DESCRIPTION
**Pull request by Dandroas**

### CODE REVIEW

File: src/Omnipay/Tests/TestCase.php [hunk 1]
```diff
@@ -25,6 +25,23 @@
     private $httpRequest;
 
     /**
+     * Converts a string to camel case
+     *
+     * @param string $str
+     * @return mixed
```
Comment (kayladnls) [inline src/Omnipay/Tests/TestCase.php]: it should return string, right?

**Comment by kayladnls** [comment]
Other than that one doc block, this looks good. Thank you.

**Comment by Dandroas** [comment]
Updated the docblock.

**Comment by barryvdh** [comment]
ucfirst + camelCase = StudlyCaps right? So isn't it more readable to have a `studly()` method that does that?
Also, why not use the [Omnipay\Common\Helper::camelCase()](https://github.com/thephpleague/omnipay-common/blob/16359241a2763e4edf7f52aead595d3f64207e32/src/Omnipay/Common/Helper.php#L18-L34) method?

**Comment by Dandroas** [comment]
Studly caps would be a better name, as for the camelCase method, I am reluctant to use it, as it is in an unreferenced repo, and I would be including it on the assumption that it would be there, without any guarantee. That having been said, I can't think of a case where you would not have it :)

**Comment by barryvdh** [comment]
Hmm, no I see it's not required. But it's using its classes to assert instanceof and also mocking them. So not sure why it wouldn't be required.

**Pull request merged**